content/en/docs/concepts/configuration/secret.md

```diff
@@ -996,28 +1006,28 @@ be available in future releases of Kubernetes.
 
 ### Protections
 
-Because `secret` objects can be created independently of the `pods` that use
+Because Secret objects can be created independently of the pods that use
 them, there is less risk of the secret being exposed during the workflow of
 creating, viewing, and editing pods.  The system can also take additional
-precautions with `secret` objects, such as avoiding writing them to disk where
+precautions with Secret objects, such as avoiding writing them to disk where
 possible.
 
-A secret is only sent to a node if a pod on that node requires it.
-Kubelet stores the secret into a `tmpfs` so that the secret is not written
-to disk storage. Once the Pod that depends on the secret is deleted, kubelet
+A Secret is only sent to a node if a Pod on that node requires it.
+`Kubelet` stores the secret into a `tmpfs` so that the secret is not written
```

Comment:
```suggestion
The kubelet stores the secret data into a `tmpfs` so that confidential information is not written
```
<hr />

Side note: data in `tmpfs` can be written to storage if your cluster has swap enabled, but that's not common.